src/EnergyPlus/UnitarySystem.cc

```diff
@@ -249,6 +249,31 @@ namespace UnitarySystems {
         if (designSpecMSHP.size() > 0) designSpecMSHP.clear();
     }
 
+    void UnitarySys::simulate(std::string const &Name,
```

Comment:
overload simulate function to accept sysOutputProvided and latOutputProvided used only for zone equipment